tweak intro on "keras updates" post (#183)
rstudio/ai-blog: _posts/2021-11-18-keras-updates/keras-updates.Rmd

---
title: "Revisiting Keras for R"
description: > 

>   It's been a while since this blog featured content about Keras for R, which may have given the impression that the project was dormant. It is high time to correct that impression.
>   Keras for R is better than ever, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks.

Reviewer: ```suggestion
  It's been a while since this blog featured content about Keras for R, so you might've thought that the project was dormant. It's not! In fact, Keras for R is better than ever, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks.
```
